### DISCUSSION TITLE
Rule validation warnings when running custom targets

### DISCUSSION THREAD

**Discussion by shawn-hurley** [konveyor/enhancements#97] [Ideas]
When a repo is checked out with rules, there could be garbage or invalid rules. These rules can be:

1. Ignored
2. Hard Error at that time. 

I think it makes sense to, by default, ignore the error and then warn the user; today we just say things were completed successfully.

**Comment by pranavgaikwad** [comment]
I think "Ignore and warn" is the right thing to do. For analyzers / providers, I think an additional validation step prior to actually running rules should be present.
